plugins/deployment_manager/test/DeploymentManagerTest.ts

```diff
@@ -215,21 +211,6 @@ describe('DeploymentManager', () => {
     // TODO: Test cache invalidation?
   });
 
-  describe('putRoots', () => {
```

Comment:
Why remove this test?

Reply:
Nobody should be calling it now on DeploymentManager (and I removed it from DeploymentManager itself). We still have the tests on roots, which DM uses directly in only one place (after it spiders).